src/shims/ApiCompatBaseline.netfx.netstandard20Only.txt

```diff
@@ -1,34 +1,12 @@
 Compat issues with assembly netstandard:
```

Comment:
Looking a lot better :)

@terrajobst here is the diff we expect to still exist between NS2.0 and net461.